### PR TITLE
Improve section on Other commit types.

### PR DESCRIPTION
**Pull request by blackxored**
* Specify they have no effect on semver. 
* Angular no longer allows "chore", I know, I know...

### CODE REVIEW

**Comment by stevemao** [comment]
Thanks for the PR @blackxored , I'm not sure if we will diverge from the angular convention and remove `chore`. But I'm happy to add the new types EG: `ci` and `build` etc. For this PR, could you remove the deletion of `chore`?

**Comment by damianopetrungaro** [comment]
Agree with @stevemao.
imho chore it's ok, we can also use a specs that does not follow strictly the angular one 😄  (this is a specs "inspired by", not "like" the one used for the angular project)

File: index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ consumers of your library:
   with [`MINOR`](http://semver.org/#summary) in semantic versioning).
 3. **BREAKING CHANGE:** a commit that has the text `BREAKING CHANGE:` at the beginning of its optional body or footer section introduces a breaking API change (correlating with [`Major`](http://semver.org/#summary) in semantic versioning). A breaking change can be
   part of commits of any _type_. e.g., a `fix:`, `feat:` & `chore:` types would all be valid, in addition to any other _type_.
-4. Others: commit _types_ other than `fix:` and `feat:` are allowed, for example [the Angular convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) recommends `docs:`, `style:`, `refactor:`, `perf:`, `test:`, `chore:`, but these tags are not mandated by the conventional commits specification.
+4. Others: commit _types_ other than `fix:` and `feat:` are allowed, for example [the Angular convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) recommends `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others, but these tags are not mandated by the conventional commits specification, and have no effect in semantic versioning.
```
Comment (damianopetrungaro) [inline index.md]: This is not true, a refactor may introduce BC and effect the semver

Comment (stevemao) [reply index.md]: I think refactor *can* introduce BREAKING CHANGE. But only internal. For software that relies on the private API it could break.

Comment (stevemao) [reply index.md]: In fact, you could introduce BREAKING CHANGE with any type. So this should be:

```
and have no effect in semantic versioning if there's no BREAKING CHANGE.
```

Comment (damianopetrungaro) [reply index.md]: Yup, @stevemao agree with your suggestion!

**Comment by blackxored** [comment]
While I agree with you all, I believe the effect of breaking changes has
already been explained above, and I’m trying to differentiate commit types
that implicitly do (like feat and fix), does that make sense?

-- 
Best Regards,

Adrian Perez
https://adrianperez.codes

**Comment by damianopetrungaro** [comment]
@blackxored the point is that is not explicit, specs MUST be explicit.
All the other things it's ok for me

**Comment by bcoe** [comment]
@blackxored sorry for the very slow turn around on my part; @blackxored I appreciate the update. Perhaps we could update the language a bit to continue recommending the type `chore:` which I think is a _great_ verb, also @damianopetrungaro recommends the type `improvement` which I think is also a good extension to the spec.

**Comment by damianopetrungaro** [comment]
@blackxored are you busy?
If you want I can do this PR for you, let me know, so we can think about other improvements.

**Comment by blackxored** [comment]
I’ll take a look at it over the weekend. But if you want to address it
before that I wouldn’t oppose.

-- 
Best Regards,

Adrian Perez
https://adrianperez.codes

**Comment by damianopetrungaro** [comment]
@blackxored No worries, if I won't see you after the weekend I'll prepare it for you ;)

**Comment by damianopetrungaro** [comment]
@blackxored 

> recommends the type improvement which I think is also a good extension to the spec.

**Comment by blackxored** [comment]
@damianopetrungaro Not sure what you meant

**Comment by damianopetrungaro** [comment]
@blackxored [here](https://github.com/conventional-changelog/conventionalcommits.org/pull/25#issuecomment-373791944)

**Comment by blackxored** [comment]
Still unsure, bare with me. `chore` was added back by referencing `config-conventional` which does include it at the moment. As for `improvement` I'm not really sure I've seen this anywhere, is this about adding it? In any case isn't it covered by `and others`?

**Comment by damianopetrungaro** [comment]
The title is `Improve section on Other commit types` so let's improve the Other section adding `improvement` too.

Has been discussed with @bcoe in #32 

Could you please add it in the Other section explaining the difference?

> **improvement:** a commit of the _type_ `improvement` improve a current implementation without adding a new feature or fixing a bug

**Comment by damianopetrungaro** [comment]
@blackxored  any update? Just missing a couple things for merge it :D

**Comment by blackxored** [comment]
Would the latest commit do it?

**Comment by damianopetrungaro** [comment]
Please, @bcoe  let me know if you prefer to reword it or change other things.

Good one @blackxored!

**Comment by damianopetrungaro** [comment]
Ping @bcoe  @stevemao

**Comment by bcoe** [comment]
@damianopetrungaro this sounds good to me, want to go ahead and merge if you're feeling good about it?

**Comment by damianopetrungaro** [comment]
@blackxored  great job man!
Sorry for the delay but finally we did it 😄

**Pull request merged**